scripts/test_django.py

```diff
@@ -0,0 +1,702 @@
+#!/usr/bin/env python3
+"""
+Run tests against Django Servers
+"""
+import sys
+import typing
+import os
+import subprocess
+import multiprocessing
+import re
+
+from pathlib import Path
+from argparse import ArgumentParser
+
+from datetime import datetime
+
+APPLICATION_ROOT = Path(__file__).parent.parent.resolve()
+"""
+The root of this application. Path(__file__) will be ./DMOD/scripts/test_django.py, 
+Path(__file__).parent will be ./DMOD/scripts, so you have to hit Path(__file__).parent.parent 
+to hit the root, i.e. ./DMOD
+"""
+
+DEFAULT_ROOT = str(APPLICATION_ROOT / "python")
+"""The root of the python libraries and services"""
+
+MANAGE_MARKER = "os.environ.setdefault('DJANGO_SETTINGS_MODULE'"
```

Comment:
Is there a missing closing paren here?

Reply:
```suggestion
MANAGE_MARKER = "os.environ.setdefault('DJANGO_SETTINGS_MODULE')"
```

Reply:
That won't work - the marker is supposed to match on lines like `x = os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'something.settings')`. Cutting it off with the `)` character will prevent it from matching on what it needs to match on.